src/context/revoltjs/util.tsx

```diff
@@ -47,3 +49,18 @@ export function getChannelName(
 
     return <>{channel.name}</>;
 }
+
+// todo: please make a better way to do this
+// server.orderedChannels or something
+export function serverChannels(server: Server, client: Client) {
```

Comment:
Just a note, `Server.client` also provides the Client so it does not need to be passed in.